File: 02-flightservice/src/main/java/com/cloudxplorer/flightservice/messaging/InventoryEvent.java
package com.cloudxplorer.flightservice.messaging;

public record InventoryEvent(
    String eventId,
    String eventType,
    String occurredAt,
    String flightId,
    String holdId,
    String bookingId,
    int seatCount,
    String actorType,
    String userId,
    String status,
    String reason,
    String source) {}

File: 03-bookingservice/Messaging/InventoryEvent.cs
namespace bookingservice.Messaging;

public record InventoryEvent(
    string EventId,
    string EventType,
    string OccurredAt,
    string FlightId,
    string HoldId,
    string BookingId,
    int SeatCount,
    string ActorType,
    string UserId,
    string Status,
    string Reason,
    string Source);
